subjects/devops/credentials_searches/README.md

```diff
@@ -0,0 +1,64 @@
+## credentials_searches
+
+### Instructions
+
+Create a file `credentials_searches.py` that contains a function `credentials_search` which takes no parameters and searches for the keys `password` and `secret` in a file called `logs.json`.
+
+- If both keys are found, the function should create a new json file named 'credentials.json' and save the values of those keys in it.
```

Comment:
Just to be sure: all the files mentioned in this bullet points are highlighted using the normal ' instead of the `

Reply:
Thanks, corrected.